fetch/api/response/multi-globals/url-parsing.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<title>Response.redirect URL parsing, with multiple globals in play</title>
+<link rel="help" href="https://fetch.spec.whatwg.org/#dom-request">
```

Comment:
Thanks, fixed. I think it's clearer to have separate test cases, but am happy to change if you'd prefer it that way.